solutions/gold/usaco-696.mdx

```diff
@@ -306,10 +307,122 @@ int main() {
 </CPPSection>
 </LanguageSection>
 
-<IncompleteSection>
+## Solution 3: HLD
 
-Add Centroid Decomp/HLD Solution
+**Time Complexity:** $\mathcal{O}(N\log ^2N)$
+
+Sort the nodes from largest to smallest value, do path updates from each node to the root.
```

Comment:
i get what you mean but like, maybe make the wording a bit more specific 

like: "Sort the nodes in descending order of proficiency. When adding in each node, we do a path update from the root to this node." (except be specific with what type of path update we do)